From e867d1c8728a9d7ed4b42581a5b9e4ae50c06b34 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 13 Jul 2022 09:59:39 +0300
Subject: [PATCH] [SW-86607] gaudi3: fix preboot text base address

Preboot is now being copied by the SPI DMA from the flash memory,
SPI DMA requires the fullchip address.

Change-Id: I52826061db8846196ef94256808688d561ee4bea
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/board/habanalabs/gaudi3_arc/post-image-spi.sh b/board/habanalabs/gaudi3_arc/post-image-spi.sh
index 78e069f..8aa6646 100755
--- a/board/habanalabs/gaudi3_arc/post-image-spi.sh
+++ b/board/habanalabs/gaudi3_arc/post-image-spi.sh
@@ -29,8 +29,7 @@
 #TODO: Fix for gaudi3
 PREBOOT_OFFSET=0x80000
 PREBOOT2_OFFSET=0xB0000
-#TODO: Fix for gaudi3
-PREBOOT_TEXT_BASE=0x1D000000
+PREBOOT_TEXT_BASE=0x0200FFFFFD000000
 
 EROM_PCIE_BINFILE=${EROM_PCIE_BINFILE:-erom_pcie.bin}
 EROM_RECOVERY_BINFILE=${EROM_RECOVERY_BINFILE:-erom_recovery.bin}
